base_miner/deepfake_detectors/ucf_detector.py

```diff
@@ -38,8 +38,6 @@ class UCFDetector(DeepfakeDetector):
     
     def __init__(self, model_name: str = 'UCF', config: str = 'ucf.yaml', cuda: bool = True):
         super().__init__(model_name, config, cuda)
-        if self.gate_name: self.gate = GATE_REGISTRY[self.gate_name]()
-        else: self.gate = None
```

Comment:
Hierarchically speaking, gates sit above UCF within CAMO, and should accordingly pass the preprocessed image into UCF, rather than being made a member of the class

Reply:
I'd reframe "using gates only in CAMO" to "Using gates in any framework that uses gating" - but yes. Deploying a standalone model with a gate doesn't really make sense.